### ISSUE TITLE
jdownloader container in start-crash loop

### ISSUE THREAD

**Issue by Orbocom** [jlesage/docker-jdownloader-2#125]
After updating jdownloader to version 22.11.2 versions, the container starts up, seems to crash but remains "running". However immediately after start  jdownloader becomes inaccessible. 

Previous versions of dockerized jdownloader worked well.

The container was created using docker-compose version 1.29.2 on a QNAP TS-451 (QTS version 5.0.1 build 20221022)

docker-compose.yml:

  jdownloader-2:
    image: jlesage/jdownloader-2
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Amsterdam
    ports:
      - "5800:5800"
      - "3129:3129"
    volumes:
      - "/share/CACHEDEV1_DATA/appdata/jdownloader-2:/config:rw"
      - "/share/CACHEDEV1_DATA/Download/completed:/output:rw"
    restart: unless-stopped

Below you may find the log: 

 [cont-env    ] TAKE_CONFIG_OWNERSHIP: loading...
[cont-env    ] XDG_CACHE_HOME: loading...
[cont-env    ] XDG_CONFIG_HOME: loading...
[cont-env    ] XDG_DATA_HOME: loading...
[cont-env    ] XDG_RUNTIME_DIR: loading...
[cont-env    ] container environment variables initialized.
[cont-secrets] loading container secrets...
[cont-secrets] container secrets loaded.
[cont-init   ] executing container initialization scripts...
[cont-init   ] 10-certs.sh: executing...
[cont-init   ] 10-certs.sh: terminated successfully.
[cont-init   ] 10-check-app-niceness.sh: executing...
[cont-init   ] 10-check-app-niceness.sh: terminated successfully.
[cont-init   ] 10-cjk-font.sh: executing...
[cont-init   ] 10-cjk-font.sh: terminated successfully.
[cont-init   ] 10-clean-logmonitor-states.sh: executing...
[cont-init   ] 10-clean-logmonitor-states.sh: terminated successfully.
[cont-init   ] 10-clean-tmp-dir.sh: executing...
[cont-init   ] 10-clean-tmp-dir.sh: terminated successfully.
[cont-init   ] 10-fontconfig-cache-dir.sh: executing...
[cont-init   ] 10-fontconfig-cache-dir.sh: terminated successfully.
[cont-init   ] 10-init-users.sh: executing...
[cont-init   ] 10-init-users.sh: terminated successfully.
[cont-init   ] 10-jwm.sh: executing...
[cont-init   ] 10-jwm.sh: terminated successfully.
[cont-init   ] 10-nginx.sh: executing...
[cont-init   ] 10-nginx.sh: terminated successfully.
[cont-init   ] 10-set-tmp-dir-perms.sh: executing...
[cont-init   ] 10-set-tmp-dir-perms.sh: terminated successfully.
[cont-init   ] 10-vnc-password.sh: executing...
[cont-init   ] 10-vnc-password.sh: terminated successfully.
[cont-init   ] 10-web-data.sh: executing...
[cont-init   ] 10-web-data.sh: terminated successfully.
[cont-init   ] 10-x11-unix.sh: executing...
[cont-init   ] 10-x11-unix.sh: terminated successfully.
[cont-init   ] 10-xdg-runtime-dir.sh: executing...
[cont-init   ] 10-xdg-runtime-dir.sh: terminated successfully.
[cont-init   ] 15-install-pkgs.sh: executing...
[cont-init   ] 15-install-pkgs.sh: terminated successfully.
[cont-init   ] 55-jdownloader2.sh: executing...
[cont-init   ] 55-jdownloader2.sh: terminated with error 1.
[init        ] container is starting...
[cont-env    ] loading container environment variables...
[cont-env    ] APP_NAME: loading...
[cont-env    ] DISPLAY: executing...
[cont-env    ] DISPLAY: terminated successfully.
[cont-env    ] DISPLAY: loading...
[cont-env    ] DOCKER_IMAGE_PLATFORM: loading...
[cont-env    ] DOCKER_IMAGE_VERSION: loading...
[cont-env    ] GTK_THEME: executing...
[cont-env    ] GTK_THEME: terminated successfully.
[cont-env    ] GTK_THEME: loading...
[cont-env    ] HOME: loading...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: executing...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: terminated successfully.
[cont-env    ] INSTALL_PACKAGES_INTERNAL: loading...
[cont-env    ] QT_STYLE_OVERRIDE: executing...
[cont-env    ] QT_STYLE_OVERRIDE: terminated successfully.
[cont-env    ] QT_STYLE_OVERRIDE: loading...
[cont-env    ] TAKE_CONFIG_OWNERSHIP: loading...
[cont-env    ] XDG_CACHE_HOME: loading...
[cont-env    ] XDG_CONFIG_HOME: loading...
[cont-env    ] XDG_DATA_HOME: loading...
[cont-env    ] XDG_RUNTIME_DIR: loading...
[cont-env    ] container environment variables initialized.
[cont-secrets] loading container secrets...
[cont-secrets] container secrets loaded.
[cont-init   ] executing container initialization scripts...
[cont-init   ] 10-certs.sh: executing...
[cont-init   ] 10-certs.sh: terminated successfully.
[cont-init   ] 10-check-app-niceness.sh: executing...
[cont-init   ] 10-check-app-niceness.sh: terminated successfully.
[cont-init   ] 10-cjk-font.sh: executing...
[cont-init   ] 10-cjk-font.sh: terminated successfully.
[cont-init   ] 10-clean-logmonitor-states.sh: executing...
[cont-init   ] 10-clean-logmonitor-states.sh: terminated successfully.
[cont-init   ] 10-clean-tmp-dir.sh: executing...
[cont-init   ] 10-clean-tmp-dir.sh: terminated successfully.
[cont-init   ] 10-fontconfig-cache-dir.sh: executing...
[cont-init   ] 10-fontconfig-cache-dir.sh: terminated successfully.
[cont-init   ] 10-init-users.sh: executing...
[cont-init   ] 10-init-users.sh: terminated successfully.
[cont-init   ] 10-jwm.sh: executing...
[cont-init   ] 10-jwm.sh: terminated successfully.
[cont-init   ] 10-nginx.sh: executing...
[cont-init   ] 10-nginx.sh: terminated successfully.
[cont-init   ] 10-set-tmp-dir-perms.sh: executing...
[cont-init   ] 10-set-tmp-dir-perms.sh: terminated successfully.
[cont-init   ] 10-vnc-password.sh: executing...
[cont-init   ] 10-vnc-password.sh: terminated successfully.
[cont-init   ] 10-web-data.sh: executing...
[cont-init   ] 10-web-data.sh: terminated successfully.
[cont-init   ] 10-x11-unix.sh: executing...
[cont-init   ] 10-x11-unix.sh: terminated successfully.
[cont-init   ] 10-xdg-runtime-dir.sh: executing...
[cont-init   ] 10-xdg-runtime-dir.sh: terminated successfully.
[cont-init   ] 15-install-pkgs.sh: executing...
[cont-init   ] 15-install-pkgs.sh: terminated successfully.
[cont-init   ] 55-jdownloader2.sh: executing...
[cont-init   ] 55-jdownloader2.sh: terminated with error 1.

What would you advice to solve the issue?
Thanks in advance.

**Comment by jlesage** [comment]
This version has an issue.  Make sure to use the latest version (22.12.1).

**Comment by modem** [comment]
I see the same issue. I'm in the latest version, just re-created the container.
One thing I noticed is that the ownership of the file `cfg/org.jdownloader.settings.GraphicalUserInterfaceSettings.lastframestatus.json` changes to **root** and it's permissions change to **600** instead of 644 that the remaining files on the cfg folder have.
Even if I restore the file permissions and ownership, it changes again when I start the container.

**Comment by jlesage** [comment]
Can you provide the output of `docker inspect <container name>` ?

**Comment by modem** [comment]
Fixed now. Thanks to the command you requested I noticed I was still using v22.11.2. I guess QNAP Container Station was using a cached version when I re-created the container, so I deleted everything related to jDownloader com the Container Station Images screen and re-created the container, now with a different output information than before:
`Unable to find image 'jlesage/jdownloader-2:latest' locally
latest: Pulling from jlesage/jdownloader-2
ca7dd9ec2225: Already exists
4f4fb700ef54: Pull complete
4d4c8e705a71: Pull complete
d89847f50f4f: Pull complete
919853a2bdbc: Pull complete
ffe60bce93d7: Pull complete
a320e8997b85: Pull complete
ce5d22580ff1: Pull complete
d1459515f3d6: Pull complete
e8824480fe87: Pull complete
7b740fa603e4: Pull complete
e85aaa9c02f0: Pull complete
f7cc8b46b04b: Pull complete
06405706e102: Pull complete
f94fa393921e: Pull complete
b07f7741b7c3: Pull complete
920f5223f5bd: Pull complete
e47fb3503c4c: Pull complete
65fbd73b8826: Pull complete
4081d1ef9cd2: Pull complete
72952cdbd93b: Pull complete
8bedd8eb8807: Pull complete
c77dd1096ffe: Pull complete
6bf8e9aa4974: Pull complete
c8c978e5feed: Pull complete
ecc6e2c8eae3: Pull complete
53b7da49ab60: Pull complete
1fbbcfd463ee: Pull complete
7e2c76dd87db: Pull complete
fdfb2ef2f9b0: Pull complete
Digest: sha256:42c649d86b5816c066ddfaeacf87690b4625a47e8687dcd60eca8235423dc442
Status: Downloaded newer image for jlesage/jdownloader-2:latest
a0a338bc7b99dd37c14616e987eadb434b66df8ed15d30b399a94a445ef9a93c
`
Now it's running.
I assumed that when I re-created the container the latest version was being used, it seems not. Sorry for that.
